### ISSUE TITLE
"Something unexpected happened" when navigating to a url that ends in `#`

### ISSUE THREAD

**Issue by chrisvariety** [brillout/vite-plugin-ssr#28]
It seems like any URL ending in `#` will easily trigger this error if you `useClientRouter`, like `http://localhost:3000/signup#`. Don't see any negative impact really, just saw the console message and figured I'd let ya know!

I'm on v0.1.0-beta.26, vite 2.1.4.

```
Uncaught (in promise) Error: [vite-plugin-ssr][Internal Error] Something unexpected happened. Please open a new issue at https://github.com/brillout/vite-plugin-ssr/issues/new and include this error stack.
    at newError2 (newError.ts:18)
    at assert (assert.ts:20)
    at getUrl (getUrl.client.ts:13)
    at fetchAndRender (useClientRouter.client.ts:78)
    at useClientRouter.client.ts:55
    at useClientRouter.client.ts:219
```

**Comment by brillout** [comment]
Fix released in `v0.1.0-beta.27`.

**Comment by brillout** [comment]
Yes you were right; it was just a wrong assertion :-)

**Comment by chrisvariety** [comment]
@brillout FYI, this is still happening on the `/#` path. Let me know if you want me to open a new issue!

```
newError.ts:18 Uncaught Error: [vite-plugin-ssr][Internal Error] Something unexpected happened. Please open a new issue at https://github.com/brillout/vite-plugin-ssr/issues/new and include this error stack.
    at newError2 (newError.ts:18)
    at assert (assert.ts:19)
    at getUrl (getUrl.client.ts:15)
    at getUrlPathname (getUrl.client.ts:22)
    at navigationState.client.ts:3
newError2 @ newError.ts:18
assert @ assert.ts:19
getUrl @ getUrl.client.ts:15
getUrlPathname @ getUrl.client.ts:22
(anonymous) @ navigationState.client.ts:3
```

version 0.1.0-beta.30, vite 2.1.4

**Comment by brillout** [comment]
Fix released in `0.1.0-beta.31`.

**Comment by brillout** [comment]
We'll soon be out of beta btw :-). One last breaking change is coming.
